manifest.json

```diff
@@ -1009,6 +1009,36 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/ubuntu/21.10/helix/arm64v8",
+              "os": "linux",
+              "osVersion": "hirsute",
+              "tags": {
+                "ubuntu-21.10-helix-arm64v8-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
+              },
+              "variant": "v8"
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/ubuntu/21.10/helix/amd64",
+              "os": "linux",
+              "osVersion": "hirsute",
+              "tags": {
+                "ubuntu-21.10-helix-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "ubuntu-21.10-helix-amd64": {
+                    "isLocal": true
```

Comment:
`"isLocal": true` is only needed if there is another Dockerfile dependent on the image produced by this Dockerfile. What this does is create a tag for the built image but this tag is not published (hence, "isLocal"). By having it tagged locally, it allows other dependent Dockerfiles to reference this tag in their `FROM` instruction. This is useful because the other tag defined here has variables in it which aren't known ahead of time. But anyway, since there is no such dependent Dockerfile, this can be removed.

Reply:
Makes sense!